detectron2/export/patch_instance.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import importlib.util
+import os
+import sys
+import tempfile
+import torch
+
+from detectron2.structures import Boxes, Instances  # noqa F401
```

Comment:
add link to https://github.com/pytorch/pytorch/issues/38964 if related

Reply:
done